modules/util/util.go

```diff
@@ -63,3 +64,35 @@ func Min(a, b int) int {
 func IsEmptyString(s string) bool {
 	return len(strings.TrimSpace(s)) == 0
 }
+
+// NormalizeEOL will convert Windows (CRLF) and Mac (CR) EOLs to UNIX (LF)
+func NormalizeEOL(input []byte) []byte {
+	tmp := make([]byte, len(input))
```

Comment:
```suggestion
	if bytes.IndexByte(input, '\r') == -1 {
		return input
	}
	tmp := make([]byte, len(input))
```

Reply:
This makes it much faster for small unix files - but does slightly slow things down on big windows/mac files